src/acceptance-test/java/org/zalando/nakadi/webservice/utils/TestStreamingClient.java

```diff
@@ -117,11 +115,8 @@ public TestStreamingClient start() {
 
     public boolean close() {
         if (running) {
-            try {
-                connection.disconnect();
-            } finally {
-                running = false;
-            }
+            running = false;
+            connection.disconnect();
```

Comment:
@adyach have you discussed that with @antban? I know that in his branch he changed the way of closing connection by closing connection of URL object.